sidebar.less

```diff
@@ -209,6 +209,11 @@
 		}
 	}
 	
+	.subButtons span:not(.subscribe-button) {	// RES subscribe buttons (+shortcut, +dashboard)
+		width: 80px;
+		margin: 0 0 5px; 5px;		
```

Comment:
Syntax error.

This is also redundant: a 5px right margin is already inherited. See first comment.